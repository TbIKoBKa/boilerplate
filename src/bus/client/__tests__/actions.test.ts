/**
 * @jest-environment jsdom
 */

// Actions
import { togglerCreatorAction } from '../togglers';

describe('togglers actions', () => {
    test('should set toggler', () => {
        expect(togglerCreatorAction({ type: 'isOnline', value: false })).toMatchSnapshot();
    });
});
