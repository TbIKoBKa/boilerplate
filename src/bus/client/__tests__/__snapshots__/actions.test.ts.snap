// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`togglers actions should set toggler 1`] = `
Object {
  "payload": Object {
    "type": "isOnline",
    "value": false,
  },
  "type": "SET_TOGGLER_STATE",
}
`;
